tests/pd/mock/server.rs

```diff
@@ -24,34 +26,48 @@ use kvproto::pdpb_grpc::{self, Pd};
 
 use super::mocker::*;
 
-pub struct Server {
-    server: GrpcServer,
+pub struct Server<C: PdMocker> {
+    server: Option<GrpcServer>,
+    mocker: PdMock<C>,
 }
 
-impl Server {
-    pub fn run<C>(eps_count: usize, handler: Arc<Service>, case: Option<Arc<C>>) -> Server
-    where
-        C: PdMocker + Send + Sync + 'static,
-    {
+impl Server<Service> {
+    pub fn new(eps_count: usize) -> Server<Service> {
+        let mgr = SecurityManager::new(&SecurityConfig::default()).unwrap();
```

Comment:
I think we can let `SecurityManager` derive the `Default` trait.

Reply:
Let's leave it to another PR ;-)